asmcli/asmcli

```diff
@@ -1614,6 +1619,13 @@ install_private_ca() {
     --location "${CA_LOCATION}" \
     --member "group:${WORKLOAD_IDENTITY}" \
     --role "roles/privateca.auditor"
+
+  if [[ 1 == $(grep -o ":" <<< "${CA_NAME}" | wc -l) ]]; then
+    local CERT_TEMPLATE; CERT_TEMPLATE=$(echo "${CA_NAME}" | cut -f2 -d:)
```

Comment:
@pmansour - do you for-see any issues with using ":" as a delimiter to specify both the ca-pool and the certificate template in the same variable?